midend/lib/Conversion/MatMulOptimization/CMakeLists.txt

```diff
@@ -11,6 +13,14 @@ add_mlir_library(BatchMatMulOptimization
   BatchMatMulOptimize.cpp
 )
 
+add_mlir_library(BatchMatMulTileOptimization
+  BatchMatMulTileOptimize.cpp
+)
+
+add_mlir_library(BatchMatMulSCFOptimization
+  BatchMatMulSCFOptimize.cpp
+)
```

Comment:
Do `BatchMatMulOptimize`, `BatchMatMulTileOptimize` and `BatchMatMulSCFOptimize` need to "add_mlir_library" twice? Should all three of them belong to `BatchMatMulOptimization`?